.github/workflows/ci-win.yml

```
@@ -0,0 +1,64 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  - pull_request
+  - push
+
+name: CI-Win
+
+jobs:
+  tests:
+    name: Tests
+
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        php-versions:
+          - "7.3"
+          - "7.4"
+          - "8.0"
+        coverage:
+          - pcov
+          - xdebug
+        exclude:
+          - php-versions: "8.0"
+            coverage: xdebug
+        
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with: 
+          fetch-depth: 1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: ${{ matrix.php-versions }}
+          coverage: ${{ matrix.coverage }}
+          extensions: curl, intl, mbstring, openssl, pdo_sqlite, soap          
+          ini-values: assert.exception=1, zend.assertions=1                  
+
+      - name: Restore cached composer dependencies 
+        uses: actions/cache@v1
+        with:
+          path: ~\AppData\Local\Composer
+          key: ${{ matrix.php-versions }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: |
+            ${{ matrix.php-versions }}-composer-      
```

What do you think about letting `composer` determine the cache directory instead?

```suggestion
      - name: Determine composer cache directory
        id: determine-composer-cache-directory
        run: echo "::set-output name=directory::$(php ./tools/composer config cache-dir)"
          
      - name: Restore cached composer dependencies 
        uses: actions/cache@v1
        with:
          path: ${{ steps.determine-composer-cache-directory.outputs.directory }}
          key: ${{ matrix.php-versions }}-composer-${{ hashFiles('**/composer.json') }}
          restore-keys: |
            ${{ matrix.php-versions }}-composer-      
```

---

Also see #4155.